### DISCUSSION TITLE
Allow Mods/Menders to "Request Re-review" for Beta Kata

### DISCUSSION THREAD

**Discussion by Kacarott** [codewars/codewars.com#3157] [Ideas]
Beta is a punishing environment. In most cases, if a kata isn't close to excellent on publication then it stands almost no chance of eventually being approved, even if its quality is dramatically improved. In general, beta solvers rarely ever revisit beta kata after solving. This means that if a kata changes either in quality or in difficulty, it will often have votes which do not reflect its current state.

I believe that the main problem is simply that most beta solvers aren't even aware when a kata they have solved changes. So to help solve this, I propose a new feature for Mods and Menders: **"Request Re-review"**. This would simply be a button which accepts a very small description of what has changed, and would then send a notification to all solvers of the kata to inform them of the changes. This would hopefully prompt people to re-evaluate their votes, if they feel it is deserved.

The reason I suggest it be restricted to Mods and Menders would be to prevent abuse and annoying spam. Authors who have improved/changed their kata could get the attention of a mod/mender who would decide if the change was substantial enough to warrant a re-review request.

**Comment by Blind4Basics** [comment]
note: some people might want to be able to configure their profile so that they can block those notifications. Or it could be considered as being part of the involvement in the beta process, maybe...?

**Reply by Kacarott** [reply]
I think notification handling in general is a very needed feature, but which is its own topic

**Reply by Blind4Basics** [reply]
yes, but this is also the topic here ;)
